module/const/action.mjs

```diff
@@ -324,7 +324,8 @@ export const TAGS = {
     prepare() {
       const stride = this.actor.system.movement.stride;
       const movement = this.usage.movement || this.actor.getMovementActionCost(stride);
-      this.cost.action = movement.cost;
+      if ( this.id === "move" ) this.cost.action = movement.cost;
+      else if ( this.actor.system.hasFreeMove ) this.cost.action = Math.max(0, this.cost.action - 1);
```

Comment:
Since `getMovementActionCost` is an Actor method, I think this should change to something like this where all the logic is placed into `getMovementActionCost` which is passed the `CrucibleAction` instance.
```js
const movement = this.actor.getMovementActionCost(this);
this.cost.action = movement.cost;
```